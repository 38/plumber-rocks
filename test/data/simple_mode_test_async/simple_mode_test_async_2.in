.TEXT get_data
{
	"command": "GET test_key"
}
.END

.TEXT get_key_2
{
	"command": "GET key_2"
}
.END

.TEXT get_key_1
{
	"command": "GET key_1"
}
.END

.TEXT get_key_3
{
	"command": "GET key_3"
}
.END

.TEXT get_not_exist_key
{
	"command": "GET not_exists"
}
.END

.TEXT remove_key
{
	"command": "PUT key_3"
}
.END

.TEXT get_key_3_not_exist
{
	"command": "GET key_3"
}
.END

.STOP
