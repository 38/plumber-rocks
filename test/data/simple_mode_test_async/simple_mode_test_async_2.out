get_data	{"data_out":"override"}
get_key_2	{"data_out":"2"}
get_key_1	{"data_out":"1"}
get_key_3	{"data_out":"3"}
get_not_exist_key	{"null":null}
remove_key	{"null":null}
get_key_3_not_exist	{"null":null}
